src/common/header/header.css

```diff
@@ -20,10 +20,6 @@
   }
 }
 
-.app-header-home {
-  position: fixed;
-}
-
```

Comment:
@kd100100 , If we remove this class then the header will be no longer fixed. is this change intentional? is yes then why?

Reply:
> I have verified on the live site and it's working fine. can you double-check is this issue exists?
> 
>  Screen.Recording.2022-10-26.at.10.07.07.PM.mov

@Sachin-chaurasiya, I have checked this issue on (MacOS - Chrome, Safari, Firefox and Windows - Chrome) now. This issue still exists on the live site `https://reactplay.io/`. In your recording I don't see a scrollbar, I guess you are checking on tablet. Could you please try on laptop/desktop?

Reply:
> @kd100100 , If we remove this class then the header will be no longer fixed. is this change intentional? is yes then why?

### Cause of the problem
When the heart is clicked the popup turns homepage into a non-scrollable part. So, the scrollbar disappears and since it was in `fixed` position, it expands itself to screen's width. That is what is causing the alignment changes in navigation bar.

### Solution
When the navigation bar is in `block` instead of `fixed` position like the other pages are currently, this problem doesn't seem to appear. I have attached the screen recoding of the behaviour change with inspect panel.

https://user-images.githubusercontent.com/64349912/198178871-53dc86a7-1621-420f-a272-8569bb7eb1e9.mp4

### Possible change that can be implemented
When position `fixed` is removed, the webpage goes down by 60px. We can fix it using margin.

Reply:
> @kd100100 , If we remove this class then the header will be no longer fixed. is this change intentional? is yes then why?

@Sachin-chaurasiya 

### Cause of the problem
When the heart is clicked the popup turns homepage into a non-scrollable part. So, the scrollbar disappears and since it was in `fixed` position, it expands itself to screen's width. That is what is causing the alignment changes in navigation bar.

### Solution
When the navigation bar is in `block` instead of `fixed` position like the other pages are currently, this problem doesn't seem to appear. I have attached the screen recoding of the behaviour change with inspect panel.

https://user-images.githubusercontent.com/64349912/198178871-53dc86a7-1621-420f-a272-8569bb7eb1e9.mp4

### Possible change that can be implemented
When position `fixed` is removed, the webpage goes down by 60px. We can fix it using margin.

Reply:
> I have verified on the live site and it's working fine. can you double-check is this issue exists?
> 
>  Screen.Recording.2022-10-26.at.10.07.07.PM.mov

@Sachin-chaurasiya Just found out that this issue occurs when connected to external mouse. Please check connecting to an external mouse.

Reply:
> https://react-play-git-fork-kd100100-nav-glitch-reactplayio.vercel.app/

Hello @kd100100 , I checked and it's working fine.

Reply:
> > @kd100100 , If we remove this class then the header will be no longer fixed. is this change intentional? is yes then why?
> 
> @Sachin-chaurasiya
> 
> ### Cause of the problem
> When the heart is clicked the popup turns homepage into a non-scrollable part. So, the scrollbar disappears and since it was in `fixed` position, it expands itself to screen's width. That is what is causing the alignment changes in navigation bar.
> 
> ### Solution
> When the navigation bar is in `block` instead of `fixed` position like the other pages are currently, this problem doesn't seem to appear. I have attached the screen recoding of the behaviour change with inspect panel.
> 
>  nav-glitch-inspect.mp4 
> ### Possible change that can be implemented
> When position `fixed` is removed, the webpage goes down by 60px. We can fix it using margin.

@kd100100 , We want the header to be fixed and if you check on the live site we have fixed the header.

Reply:
@Sachin-chaurasiya This issue occurs when connected to external mouse. Could you connect an external mouse and see if this issue is resolved.

Just wanted to know, why do you want it `position fixed` only on the home page?

Reply:
@kd100100 , Yes I tested with an external mouse and it's working fine.

@kd100100 , We have a header fixed on every page, please go through all the pages.

Reply:
@Sachin-chaurasiya I checked on all pages. It has `position: relative`. The following image is plays page.

<img width="1727" alt="Screenshot 2022-10-27 at 1 32 32 PM" src="https://user-images.githubusercontent.com/64349912/198227183-600479f8-ae2f-4264-98c7-58ae9b157dec.png">

Reply:
@kd100100 , Ah, got it.
But still we want header to be fixed on home page as well.

Reply:
I checked the preview https://react-play-ard3746fr-reactplayio.vercel.app/ and the header is not fixed. we can't break another thing to fix one thing.

Reply:
> I checked the preview https://react-play-ard3746fr-reactplayio.vercel.app/ and the header is not fixed. we can't break another thing to fix one thing.

I been trying to come up with a solution but not able to find. If you have one, could you suggest me?

Reply:
@kd100100 , I have verified it and it's not reproducible even with external mouse.

Reply:
Hi @Sachin-chaurasiya. Sorry for the late reply.
I have checked again and the problem still exists in my browser (`https://reactplay.io/`). But I am not sure if it has to do something with the browser settings. Shall we close this PR without merging?

Reply:
Thanks for confirmation @kd100100, I will close this PR, and you can work on other issues.
Happy coding!